### DISCUSSION TITLE
Getting error while saving "STORE INFORMATION"

### DISCUSSION THREAD

**Discussion by ashishkpaul** [saleor/saleor#10404] [Q&A] [answered]
Hello team, settings are not saved after filling **store information**
Please Help me. The error log shown bellow

![Screenshot 2022-08-17 13:43:05](https://user-images.githubusercontent.com/16308863/185069162-cd90bf7a-a85e-478a-a636-ee0994858642.png)

[pid: 7607|app: 0|req: 238/238] 223.178.208.246 () {54 vars in 858 bytes} [Wed Aug 17 08:01:06 2022] OPTIONS /graphql/ => generated 0 bytes in 1 msecs (HTTP/1.1 200) 9 headers in 396 bytes (1 switches on core 0)

[pid: 7607|app: 0|req: 239/239] 223.178.208.246 () {62 vars in 2090 bytes} [Wed Aug 17 08:01:06 2022] POST /graphql/ => generated 1129 bytes in 1322 msecs (HTTP/1.1 200) 8 headers in 375 bytes (1 switches on core 0)

{"asctime": "2022-08-17T08:01:07Z", "levelname": "INFO", "lineno": 214, "message": "A query had an error", "name": "saleor.graphql.errors.handled", "pathname": "/home/ashish/Downloads/projects/saleor/codes/saleor-api/./saleor/graphql/utils/__init__.py", "process": 7607, "threadName": "uWSGIWorker1Core0", "exc_info": "graphql.error.base.GraphQLError: Variable \"$isCloudInstance\" is never used in operation \"ShopSettingsUpdate\".", "hostname": "cloud-pc"}

{"asctime": "2022-08-17T08:01:07Z", "levelname": "WARNING", "lineno": 224, "message": "Bad Request: /graphql/", "name": "django.request", "pathname": "/home/ashish/Downloads/projects/saleor/codes/venv/venv-p3.9/lib/python3.9/site-packages/django/utils/log.py", "process": 7607, "threadName": "uWSGIWorker1Core0", "status_code": 400, "request": "<WSGIRequest: POST '/graphql/'>", "hostname": "cloud-pc"}
[pid: 7607|app: 0|req: 240/240] 223.178.208.246 () {64 vars in 3087 bytes} [Wed Aug 17 08:01:07 2022] POST /graphql/ => generated 284 bytes in 10 msecs (HTTP/1.1 400) 8 headers in 383 bytes (1 switches on core 0)

**Comment by timuric** [comment, marked as answer]
@ashishkpaul there is a PR soon to be merged that resolves this issue, expect it to be in the next release https://github.com/saleor/saleor-dashboard/pull/2228
